src/qt/radiobut.cpp

```diff
@@ -59,3 +77,44 @@ QWidget *wxRadioButton::GetHandle() const
 {
     return m_qtRadioButton;
 }
+
+void wxRadioButton::createAndJoinNewGroup( wxWindow *parent )
+{
+    m_qtButtonGroup = new QButtonGroup( parent->GetHandle() );
```

Comment:
Is there any reason why simply doing [...]` = new QButtonGroup(GetHandle())` wouldn't work? IOW set the first button of a group to be the object responsible with cleaning up the group. It would eliminate the need for explicit cleanup.

Reply:
Good spot. Doing it that way in the latest commit.